connector/sources/wpf.py

```diff
@@ -0,0 +1,182 @@
+    {"key": "296", "label": "Sao Tome and Principe"},
+    {"key": "298", "label": "Senegal"},
+    {"key": "302", "label": "Sierra Leone"},
+    {"key": "307", "label": "Somalia"},
+    {"key": "3647", "label": "South Africa"},
+    {"key": "3652", "label": "South Sudan"},
+    {"key": "311", "label": "Sri Lanka"},
+    {"key": "112", "label": "Sudan"},
+    {"key": "108", "label": "Syrian Arab Republic"},
+    {"key": "109", "label": "Tajikistan"},
+    {"key": "318", "label": "Tanzania, United Republic of"},
+    {"key": "321", "label": "Timor-Leste"},
+    {"key": "322", "label": "Togo"},
+    {"key": "3779", "label": "Tunisia"},
+    {"key": "5108", "label": "Turkey"},
+    {"key": "332", "label": "Uganda"},
+    {"key": "5071", "label": "Ukraine"},
+    {"key": "3658", "label": "Uzbekistan"},
+    {"key": "4791", "label": "Viet Nam"},
+    {"key": "110", "label": "Yemen"},
+    {"key": "345", "label": "Zambia"},
+    {"key": "346", "label": "Zimbabwe"},
+]
+
+YEAR_OPTIONS = [
+    {"key": "All", "label": "Any"},
+    {"key": "421", "label": "1993"},
+    {"key": "420", "label": "1994"},
+    {"key": "377", "label": "1995"},
+    {"key": "378", "label": "1996"},
+    {"key": "379", "label": "1997"},
+    {"key": "380", "label": "1998"},
+    {"key": "381", "label": "1999"},
+    {"key": "368", "label": "2000"},
+    {"key": "369", "label": "2001"},
+    {"key": "370", "label": "2002"},
+    {"key": "371", "label": "2003"},
+    {"key": "372", "label": "2004"},
+    {"key": "373", "label": "2005"},
+    {"key": "374", "label": "2006"},
+    {"key": "375", "label": "2007"},
+    {"key": "376", "label": "2008"},
+    {"key": "1277", "label": "2009"},
+    {"key": "2595", "label": "2010"},
+    {"key": "3464", "label": "2011"},
+    {"key": "3916", "label": "2012"},
+    {"key": "4184", "label": "2013"},
+    {"key": "4689", "label": "2014"},
+    {"key": "5006", "label": "2015"},
+    {"key": "5109", "label": "2016"},
+    {"key": "5142", "label": "2017"},
+    {"key": "5143", "label": "2018"},
+]
+
+
+@handle_connector_parse_error
+class WorldFoodProgramme(Source):
+    URL = 'https://www.wfp.org/food-security/assessment-bank'
+    title = 'WFP Assessments'
+    key = 'world-food-programme'
+    options = [
+        {
+            'key': 'tid_1',
+            'field_type': 'select',
+            'title': 'Country',
+            'options': COUNTRIES_OPTIONS
+        },
+        {
+            'key': 'tid_6',
+            'field_type': 'select',
+            'title': 'Year',
+            'options': YEAR_OPTIONS
+        },
+    ]
+
+    def fetch(self, params, page=None, limit=None):
+        results = []
+        if page:
+            params['page'] = page
+        resp = requests.get(self.URL, params=params)
+
+        soup = Soup(resp.text, 'html.parser')
+        contents = soup.find('div', {'class': 'view-content'})
+        if not contents:
+            return results, len(results)
+        # iterate and get loops
```

Comment:
Are you sure this is `loops` and not `leads`?

Reply:
Lol.